linera-base/src/identifiers.rs

```diff
@@ -51,6 +51,16 @@ impl AccountOwner {
     }
 }
 
+impl std::fmt::Debug for AccountOwner {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            AccountOwner::Reserved(value) => write!(f, "Reserved({})", value),
+            AccountOwner::Address32(address) => write!(f, "Address32({})", address),
+            AccountOwner::Address20(address) => write!(f, "Address20({})", hex::encode(address)),
```

Comment:
Could we keep the derive, `use custom_debug_derive::Debug`, and annotate the field with `#[debug(with = "hex_debug")]`?